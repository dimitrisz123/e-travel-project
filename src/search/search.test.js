import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Search from "./search";

it("renders Search correctly", () => {
	const wrapper = shallow(<Search />);
	expect(toJson(wrapper)).toMatchSnapshot();
});
