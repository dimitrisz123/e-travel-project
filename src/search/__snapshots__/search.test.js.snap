// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Search correctly 1`] = `
<input
  className="search"
  onChange={[Function]}
  placeholder="Search for movies"
  type="text"
/>
`;
